lib/web_ui/test/golden_tests/engine/compositing_golden_test.dart

```diff
@@ -2,19 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
 import 'dart:html' as html;
 import 'dart:math' as math;
 
 import 'package:test/bootstrap/browser.dart';
 import 'package:test/test.dart';
-import 'package:ui/ui.dart';
+import 'package:ui/ui.dart' as ui;
```

Comment:
I’m curious if there was a reason for this change.

Reply:
This is to disambiguate class names such as `ClipRectEngineLayer`. Both `ui` and `_engine` define these classes.

Reply:
For background, the reason we use the exact same class names is because diagnostics on the framework side rely on `runtimeType.toString`, so we can't rename the class.